src/exercises/PrioritizeBookmarksToStudy.js

```diff
@@ -0,0 +1,36 @@
+import { MAX_BOOKMARKS_PER_ARTILE } from "./ExerciseConstants.js";
+
+export default function prioritizeBookmarksToStudy(
```

Comment:
I see your point. My argument is that the BookmarksToReview only exist because of exercises otherwise it would just be a list of words. In my view, I would expect to see Bookmarks related constants in the exercises and the Reader to only handle the logic of the aspects relating to the reading experience. 

I can move it you prefer? But then what should it be called? We don't really have a file for these things for the Article.